app/App/Account/Account/Requests/TransactionRequest/TxFeeNew/index.js

```diff
@@ -101,7 +101,11 @@ class TxFee extends React.Component {
             <ClusterColumn grow={2}>
               <ClusterValue>
                 <div className='txSendingValue'>
-                  <DisplayValue type='ether' valueData={maxFee} currencySymbol={nativeCurrency.symbol} />
+                  <DisplayCoinBalance
+                    amount={maxFee.wei().value.toString()}
+                    decimals={18}
```

Comment:
The decimals are defaulted to 18 in displayValueData so you don't need to specify them